src/Cache/FileCache.php

```diff
@@ -79,19 +79,19 @@ protected function file(string $key): string
 				case '/':
 					break;
 
-				// backslashes get their own marker in the path
-				// to differentiate the cache key from one with forward slashes
+					// backslashes get their own marker in the path
+					// to differentiate the cache key from one with forward slashes
```

Comment:
We need to move these comments below the `case` statements. With the new indentation they are quite confusing.

Reply:
To be honest, these indentations are getting on my nerves 😩

Reply:
We only need to fix them once by moving the comments, then we should be fine in the long term.